packages/fsml/fsml.0.2a/opam

```diff
@@ -0,0 +1,31 @@
+opam-version: "2.0"
+synopsis:
+  "A library for describing and describing synchronous finite state machines"
+description:
+  "A library for describing and describing synchronous finite state machines"
+maintainer: ["Jocelyn Sérot <jocelyn.serot@uca.fr>"]
+authors: ["Jocelyn Sérot <jocelyn.serot@uca.fr>"]
+license: "MIT"
+homepage: "http://github.com/jserot/fsml"
+doc: "http://jserot.github.io/fsml"
+bug-reports: "jocelyn.serot@uca.fr"
+depends: [
+  "dune" {>= "2.6" & >= "2.0"}
+  "menhir" {>= "2.0"}
```

Comment:
Menhir's version scheme is of the form `YYYYMMDD`. Which version are you referring to?

Reply:
20200624

Reply:
Fixed